source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst

```diff
@@ -13,8 +13,8 @@ SDAM Monitoring Specification
 :Advisory Group: Jeff Yemin, Craig Wilson, Jesse Davis
 :Status: Approved
 :Type: Standards
-:Minimum Server Version: 2.5
-:Last Modified: 29-Jan-2020
+:Minimum Server Version: 2.4
+:Last Modified: 12-Dec-2018
```

Comment:
This date doesn't look right.

Also, consider bumping the spec version to 1.2.0 since you're introducing new SHOULD/MUST lines.